Tavisca.Bootcamp.LanguageBasics.Exercise1/Equation.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
+{
+    class Equation
+    {
+
+        public Equation(string operand1,string operand2,string operation,string result)
+        {
+            Operand1 = operand1;
+            Operand2 = operand2;
+            Operation = operation;
+            Result = result;
+        }
+
+        public string Operand1 { get; private set; }
+        public string Operand2 { get; private set; }
+        public string Operation { get; private set; }
+        public string Result { get; private set; }
+
+        public int CorrectEquationAndReturnMissingDigit()
+        {
+            EquationParts needsCorrection = GetEquationPartToBeCorrected();
+            string expectedValue;
```

Comment:
you don't need expected value later, why are you storing it in a variable?? 
use var expectedValue = some operation in per case only, so that this variable's scope will be limited to that block of code